### PR TITLE
Add Cloud Watch Log Hook

### PR DESCRIPTION
**Pull request by lpichler**
1. Change way of configuration for middleware logger
2. Added method `AddHooksTo(logger *logrus.Logger, config *appconf.SourcesApiConfi)`  which adds  _log hooks_  to `logger`
   A.  method `AddHooksTo` is used for all logging instances:  general logger, echo logger, gorm logger and middleware logger(now this logger tied to general logger)

## Links

https://issues.redhat.com/browse/RHCLOUD-15683

### CODE REVIEW

File: logger/log_writter.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package logger
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+type LogWritter struct {
```
Comment (lindgrenj6) [inline logger/log_writter.go]: Can we rename this to `LogWriter`? I think thats what you were going for unless I'm missing something (filename too).

**Review by lindgrenj6 — COMMENTED** [review]
Just one small typo I think in the struct name - but looks good to me otherwise!

File: logger/log_writter.go (hunk 1 above)
Comment (lpichler) [reply logger/log_writter.go]: sure, I fixed it 👍

File: logger/log_writter.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package logger
```
Comment (lindgrenj6) [inline logger/log_writter.go]: I hate to be that guy but the filename is still `log_writter.go` :grimacing:

Comment (lindgrenj6) [reply logger/log_writter.go]: I edited it in github since apparently I can do that. Merging....

**Review by lindgrenj6 — APPROVED** [review]

File: logger/log_writter.go (hunk 2 above)
Comment (lpichler) [reply logger/log_writter.go]: oh I am sorry I thought that the file had correct name 🤦 thanks @lindgrenj6 !

**Pull request merged**